symfony/plugins/orangehrmLeavePlugin/Api/EmployeeLeaveRequestAPI.php

```diff
@@ -68,6 +71,20 @@ class EmployeeLeaveRequestAPI extends Endpoint implements CrudEndpoint
         self::MODEL_DETAILED => LeaveRequestDetailedModel::class,
     ];
 
+    protected ?LeaveApplicationService $leaveApplicationService = null;
```

Comment:
`$leaveApplicationService` is this in use

Reply:
Removed. Done